nvmeadm/src/nvmf_discovery.rs

```diff
@@ -171,7 +171,7 @@ impl Discovery {
         let _ret = unsafe {
             convert_ioctl_res!(nix_ioctl(
                 f.as_raw_fd(),
-                u64::from(NVME_ADMIN_CMD_IOCLT).try_into().unwrap(),
+                u64::from(NVME_ADMIN_CMD_IOCLT),
```

Comment:
Since we're here ... I'm sure Jeffry won't mind if we change this to the proper spelling: NVME_ADMIN_CMD_IOCTL

Reply:
Done